pkg/provider/azure_loadbalancer.go

```diff
@@ -1561,7 +1561,7 @@ func (az *Cloud) reconcileLoadBalancer(clusterName string, service *v1.Service,
 	// reconcile the load balancer's backend pool configuration.
 	if wantLb {
 		preConfig, changed, shouldRefreshLB, err := az.LoadBalancerBackendPool.ReconcileBackendPools(clusterName, service, lb)
-		if err != nil {
+		if err != nil && !retry.HasStatusForbiddenOrIgnoredError(err) {
```

Comment:
And could you add some unit tests for newly added codes?

Reply:
Added.